tests/cli-tests/run.py

```diff
@@ -278,7 +278,8 @@ def _test_environment(self) -> typing.Dict[str, str]:
         Returns the environment to be used for the
         test subprocess.
         """
-        env = copy.copy(os.environ)
+        # We want to omit ZSTD cli flags so tests will be consistent across environments
+        env = {k: v for k, v in os.environ.items() if not k.startswith("ZSTD")}
```

Comment:
Ohhhhh I figured out why its broken... This function is missing a `return env`... Adding that fixes it.

It only works before because apparently `copy.copy(os.environ)` doesn't really copy it. And modifying `env` adds stuff to `os.environ`.

Python is fun.

Reply:
Hah, guess we came to this realization at the same time :)
Tests are running now.